### PR TITLE
Add affinity-designer

### PR DESCRIPTION
**Pull request by Chandler-Lu**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/affinity-designer.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask 'affinity-designer' do
+  version '1.8.2'
+  sha256 'bc9076159db2cb4f17e7239ec43157574a574c670b484ff07835a61732f4b23d'
+
+  # s3-eu-west-1.amazonaws.com was verified as official when first introduced to the cask
+  url "https://s3-eu-west-1.amazonaws.com/affinity-update/mac/retail/Affinity%20Designer-#{version}.app.zip"
+  appcast 'https://s3-eu-west-1.amazonaws.com/affinity-update/mac/retail/designer.xml'
+  name 'affinity-designer'
```
Comment (vitorgalvao) [inline Casks/affinity-designer.rb]: This is the name of the app, not a repeat of the token.

File: Casks/affinity-designer.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+cask 'affinity-designer' do
+  version '1.8.2'
+  sha256 'bc9076159db2cb4f17e7239ec43157574a574c670b484ff07835a61732f4b23d'
+
+  # s3-eu-west-1.amazonaws.com was verified as official when first introduced to the cask
+  url "https://s3-eu-west-1.amazonaws.com/affinity-update/mac/retail/Affinity%20Designer-#{version}.app.zip"
```
Comment (vitorgalvao) [inline Casks/affinity-designer.rb]: I don’t see a way to confirm this link without registering.

File: Casks/affinity-designer.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/affinity-designer.rb]: This is the name of the app, not a repeat of the token.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/affinity-designer.rb (hunk 2 above)
Comment (Chandler-Lu) [reply Casks/affinity-designer.rb]: I get it by packet analyzer. You can download the software in [https://store.serif.com/en-us/download/corporate/trial/](https://store.serif.com/en-us/download/corporate/trial/) and use packet analyzer to get the true url.
截屏2020-03-2107 47 08

Comment (vitorgalvao) [reply Casks/affinity-designer.rb]: Thank you, but this makes it a [walled build](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) so [we will not accept it](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#unofficial-vendorless-and-walled-builds) for now (I’m thinking of revising that rule).

**Pull request closed without merging**